prusti-viper/src/encoder/builtin_encoder.rs

```diff
@@ -59,6 +63,7 @@ impl BuiltinEncoder {
             BuiltinMethodKind::HavocBool => "builtin$havoc_bool".to_string(),
             BuiltinMethodKind::HavocInt => "builtin$havoc_int".to_string(),
             BuiltinMethodKind::HavocRef => "builtin$havoc_ref".to_string(),
+            BuiltinMethodKind::HavocF32 | BuiltinMethodKind::HavocF64 => "builtin$havoc_float".to_string(),
```

Comment:
Can the same method accept both types? We are not mangling these names, this seems like it should fail.